src/uikit/DataTable/ToolbarButtons/ColumnFilter.js

```diff
@@ -7,10 +7,14 @@ class ColumnFilter extends Component {
 
   // keep dropdown open for multi select, close on click away
   handleStateChange = changes => {
-    const { type } = changes;
-    this.setState({
-      isOpen: type === Downshift.stateChangeTypes.mouseUp ? false : true,
-    });
+    const { type, isOpen } = changes;
+    if (typeof isOpen !== 'undefined' && type === Downshift.stateChangeTypes.clickButton) {
+      this.setState({ isOpen });
+    }
+  };
+
+  handleClose = () => {
+    this.setState({ isOpen: false });
```

Comment:
lgtm but any implication of this change on existing components that comes to mind @ciaranschutte? It'd be great if you could check this out in the portal :)

Reply:
Offhand I think this actually fixes a bug in the other place it's used on `File Entity`. Having some environment issues right now, but will check asap

Reply:
This is a fix to #1180 , done on this branch since it's the only way I was able to reproduce.

I should have mentioned that in the description of this PR. :thinking:

Reply:
👍 thank you! :)